packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -2540,6 +2540,11 @@ module.exports = function (log, config) {
     };
   });
 
+  Mailer.prototype._legalDocsRedirectUrl = function (url) {
+    const parsedUrl = new URL(config.subscriptions.paymentsServer.url);
```

Comment:
hmm we could move this out to module scope and have the payment server url config value parsed just once.